src/integration/Sites.spec.ts

```diff
@@ -0,0 +1,205 @@
+import express from "express"
+import mockAxios from "jest-mock-axios"
+import request from "supertest"
+
+import { IsomerAdmin, Repo, Site, SiteMember, User } from "@database/models"
+import { generateRouter } from "@fixtures/app"
+import UserSessionData from "@root/classes/UserSessionData"
+import { mockEmail, mockIsomerUserId } from "@root/fixtures/sessionData"
+import { SitesRouter as _SitesRouter } from "@root/routes/v2/authenticated/sites"
+import { genericGitHubAxiosInstance } from "@root/services/api/AxiosInstance"
+import { GitHubService } from "@root/services/db/GitHubService"
+import { ConfigYmlService } from "@root/services/fileServices/YmlFileServices/ConfigYmlService"
+import IsomerAdminsService from "@root/services/identity/IsomerAdminsService"
+import { SitesService } from "@root/services/utilServices/SitesService"
+import { getUsersService } from "@services/identity"
+import { sequelize } from "@tests/database"
+
+const mockSite = "mockSite"
+const mockSiteId = "1"
+const mockBlockedSite = "adminOnly"
+const mockUpdatedAt = "now"
+const mockPermissions = { push: true }
+const mockPrivate = true
```

Comment:
i think we can omit this :P but if you wanna keep this, could we rename it to `mockIsPrivate`?

Reply:
Hmm i was thinking this is useful because we can check that regular users only have access to items in the site members database that include them, while admin users see all by default - I think it might be better to leave in, but I'm not sure if `mockIsPrivate` conveys that idea